apps/frontend/src/pages/login.tsx

```diff
@@ -110,7 +81,7 @@ const LoginPage: React.FC = () => {
 
     try {
       setLoading(true);
-      await processLoginResponse(loginResponse, email, password);
+      await processLoginResponse(loginResponse, loginResponse.email, password);
```

Comment:
When I wake up from a nap I'm going to check if loginResponse.email exists. It felt like email has been removed from most locations and replaced with username.